tests/test_meta/test_thresholder.py

```diff
@@ -102,3 +105,20 @@ def fit(self, X, y, sample_weight=None):
     weight = np.random.random(100)
 
     mod.fit(X, y, sample_weight=weight)
+
+
+def test_stacking_classifier():
```

Comment:
Would be good to add a docstring that explains what/why we are testing here, potentially with a link to the original issue.